examples/values_change_ntp.yaml

```diff
@@ -20,8 +20,8 @@ command:
   cat $CONFIG
   echo "\n"
 
-  echo "RESTARTING timesync SERVICE"
-  systemctl restart systemd-timesyncd
+  echo "KILLING timesync SERVICE"
```

Comment:
I think `RESTARTING` was still a legit term for that. I'd only explain that we can't use `systemctl` and as sending `SIGHUP` as a workaround in the comment.